independent-publisher-connectors/Pushover/apiProperties.json

```diff
@@ -0,0 +1,40 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Token",
+          "description": "The API Token for this api",
+          "tooltip": "Provide your API Token",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",
```

Comment:
@glenhutson for all our independent publisher connector we ask to set the "iconBrandColor" property to "#da3b01". Please see our template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json).

Reply:
Icon color updated.